### PR TITLE
JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by jorgeabrahan**
✅ Setup project with webpack and JEST
✅ Create a project homepage
✅ Create project comments popup
✅ Retrieve data from both APIS on the homepage and also in the comments popup
✅ Manage project counters (Amount of meals in each category, amount of likes, amount of comments)
✅ Merge conflicts solved
✅ Create project tests with JEST DOM

### CODE REVIEW

File: src/css/modal.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+#add-button {
+  display: block;
+  width: 100%;
+  padding: 0.3rem 0.6rem;
+  border: 3px solid black;
+  box-shadow: 3px 3px 0 0 black;
+  transition: transform 100ms ease, box-shadow 100ms ease;
+}
```
Comment (AbayomiOlaoye) [inline src/css/modal.css:57]: - [ ] Good job, Team! you guys are invincible, I only hope you can check what I found out here to further improve your user experience. The comment pop up is showing up in to places and the second cannot be exited - at the bottom (footer section). Please check [this video](https://drive.google.com/file/d/1YMBRgrCFusk_h4by5ygo7EsF7buQlVta/view?usp=sharing)

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    <h2>Recipe</h2>
+    <p class="instructions"></p>
+    <h2 class="comments-title"></h2>
+    <ul class='comments-container'>
+    </ul>
+    <form class="input-section">
+      <h2>Add a comment</h2>
+      <input type="text" placeholder="Your name" class="user" size="25">
+      <input type="text" placeholder="Your insights" class="comment" size="50">
+      <input type="button" id="add-button" value="Comment">
```
Comment (AbayomiOlaoye) [inline src/index.html]: - [ ] Great job! Your project looks great but I think you should try to improve the quality of your user experience as well as the validity of user input by implementing some form validations whether from the client (HTML) or using JavaScript. Right now, I was able to submit an empty comment, which I think is not a best practice.

Here is a [demo](https://drive.google.com/file/d/1UaecUDPZunaEbg0FDtcjSm55yns6RpmF/view?usp=sharing) of my finding.

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @jorgeabrahan  and @edahigure :wave, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Great API implementation :heavy_check_mark: 
- Great designs :heavy_check_mark: 

## Suggested changes
- [ ] _Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 🎉 🎉 🎉 
Hi @jorgeabrahan & @edahigure , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] Good UI design :white_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 

## Optional suggestions
- [ Optional] You implemented a very nice `UI` for your `App`. I would like to suggest that you add a screenshot of the `UI design` to your readme file for the `world` to see.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**